packages/node/src/store.ts

```diff
@@ -245,7 +245,9 @@ export class Store {
     );
 
     if (!appInstanceProposal) {
-      throw Error(NO_PROPOSED_APP_INSTANCE_FOR_APP_INSTANCE_ID(appInstanceId));
+      throw new Error(
```

Comment:
`new` is not needed

Reply:
It is the convention we follow

Reply:
Do another search. We mostly don't use `new`.